cypress.config.ts

```diff
@@ -0,0 +1,10 @@
+import { defineConfig } from 'cypress';
+
+export default defineConfig({
+  e2e: {
+    setupNodeEvents(on, config) {
+      // implement node event listeners here
+    },
+    baseUrl: 'http://localhost:5173',
```

Comment:
what do you think about using `const API_URL = import.meta.env.VITE_API_URL;` here?